online/shipping-app-backend/quota.yaml

```diff
@@ -1,7 +0,0 @@
-kind: ResourceQuota
-apiVersion: v1
-metadata:
-  name: pod-quota
```

Comment:
Do we want this to be allowed? Since the quota will be renamed.

Reply:
cannot have a resource without name